docs/getting-started.rst

```diff
@@ -4,101 +4,244 @@
+
+* **Alignment**
+Three different alignments can be chosen and visualised in this tab:
+
+1. The **seed** alignment shows the multiple sequence alignment used to create the HMM model in Pfam. This is a representative set of sequences of the family and it normally has a relatively short number of protein sequences.
+
+2. The **full** alignment shows all the protein sequences from the Uniprot Reference proteomes that match this model.
+
+3. The **uniprot** alignment includes all the protein sequences matched by this Pfam model in the whole Uniprot database.
+
+The colour coding of the alignment can be customised through the options available in the **Colors** section.
 
-    * text fields in Pfam entries, e.g. family descriptions
-    * UniProt sequence entry description and species fields
-    * HEADER and TITLE fields from PDB entries
-    * Gene Ontology IDs and terms
-    * InterPro entry abstracts
+* **Curation**
+This tab is divided into two subsections:
+
+1. In the first section, you can see details about Pfam curators and Sequence ontology.
+
+2. The second section displays the HMM building command used to generate the HMM profile defining the signature.
 
-Each Pfam-A entry is listed only once in the results table, although it might have been found in more than one area of the database.
 
 Searching a protein sequence against Pfam
 =========================================
 
-Searching a protein sequence against the Pfam library of HMMs will enable you to find out the domain architecture of the protein. If your protein is present in the version of UniProt, NCBI Genpept or the metagenomic sequence set that we used to make the current release of Pfam, we have already calculated its domain architecture. You can access this by entering the sequence accession or ID in the 'view a sequence' box on the Pfam homepage.
+Searching a protein sequence against the Pfam library of HMMs will enable you to find out the domain architecture of the protein, and thus what its potential function might be. If your protein is present in the version of UniProt used to make the current release of InterPro, we have already calculated its domain architecture. You can access this by entering the Uniprot sequence identifier in any of the Search boxes mentioned above.
```

Comment:
Add a reference to the Search section at the end of the paragraph. To do so add the following, before the search section title : `.. _search:
`
and add a reference to it in the Search ways: :ref:`search`